decode/neuralfitter/train/live_engine.py

```diff
@@ -81,23 +81,28 @@ def live_engine_setup(param_file: str, cuda_ix: int = None, debug: bool = False,
 
     """Experiment ID"""
     if not debug:
-        experiment_id = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S") + '_' + socket.gethostname()
-
-        if log_comment:
-            experiment_id = experiment_id + '_' + log_comment
-
+        if param.InOut.checkpoint is None:
+            experiment_id = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S") + '_' + socket.gethostname()
```

Comment:
Should we make a new tensorboard experiment or continue the old one?

Reply:
I think that the main scenario for using the checkpoint is resuming training after it crashed / got aborted by mistake. 
So using the same logdir makes sense imo. For potential future functionality where you use pretrained networks you included the 'model_init' right?

Reply:
Yeah, on the other hand tensorboard is not stable anyways, so maybe trying to append an old tensorboard file is not a good idea